pkg/scheduler/metrics/metrics.go

```diff
@@ -23,90 +23,90 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
-const schedulerSubsystem = "scheduler"
+const (
+	SchedulerSubsystem = "scheduler"
+	SchedulingLatencyName = "scheduling_latencies_summary"
+
+	OperationLabel = "opertion"
```

Comment:
operation

Reply:
Also can we rename this variable to `OperationType` and change the below ones to simply `Binding`, `SchedulingAlgorithm`, etc? Will make the meaning more clearer imo.